packages/amplify-category-api/src/provider-utils/awscloudformation/utils/rds-resources/database-resources.ts

```diff
@@ -5,9 +5,18 @@ import { SSMClient } from './ssmClient';
 import { ImportedRDSType } from '@aws-amplify/graphql-transformer-core';
 import { MySQLDataSourceAdapter, Schema, Engine, DataSourceAdapter, MySQLDataSourceConfig } from '@aws-amplify/graphql-schema-generator';
 import { printer } from '@aws-amplify/amplify-prompts';
+import { DeleteFunctionCommand, LambdaClient } from '@aws-sdk/client-lambda';
+import { DeleteRoleCommand, IAMClient } from '@aws-sdk/client-iam';
 
 const secretNames = ['database', 'host', 'port', 'username', 'password'];
 
+export const getVpcMetadataLambdaName = (appId: string, envName: string): string => {
+  if (appId && envName) {
+    return `${appId}-rds-schema-inspector-${envName}`;
+  }
+  throw new Error("AppId and environment name are required to generate the schema inspector lambda.");
+};
```

Comment:
This null check isn't useful since these are both required params, right? Are the types wrong?